File: Edabit/php/three.php
<?php

// Create a function that accepts an array and returns the last item in the array

function getLastItem($arr)
{
  return $arr[sizeof($arr)-1];
}

$array = ['test', 'test2', 'test3'];

echo sprintf("%s\n", getLastItem($array));

// NOTES: we could have also used end();

?>

File: Edabit/java/three.java
/*
   
Create a method that accepts an array and returns the last item in the array.

*/

public class Program
{
  public static int getLastItem(int[] nums)
  {
    return nums[nums.length-1];
  }

  public static void main(String[] args)
  {
    System.out.printf("%d\n", getLastItem(new int[]{1, 2, 3}));
  }
}
